images/airflow/2.10.1/run.sh

```diff
@@ -101,4 +105,10 @@ export MWAA__CORE__TASK_MONITORING_ENABLED
 export MWAA__CORE__TERMINATE_IF_IDLE
 export MWAA__CORE__MWAA_SIGNAL_HANDLING_ENABLED
 
-$CONTAINER_RUNTIME compose up
+if [ "$COMMAND" == "resetdb" ]; then
+    $CONTAINER_RUNTIME compose -f docker-compose-resetdb.yaml up --abort-on-container-exit
+elif [ "$COMMAND" == "test-requirements" ] || [ "$COMMAND" == "test-startup-script" ]; then
+    $CONTAINER_RUNTIME compose -f docker-compose-commands.yaml up "$COMMAND" --abort-on-container-exit
```

Comment:
Just a minor comment about the naming of the commands file. It is not following the current convention. Maybe we can have a separate file for each test commands like `docker-compose-resetdb.yaml`. Or, maybe we can rename this file as `docker-compose-test-commands.yaml` to suggest that this file is for test scripts only. Right now, it is a bit too generic.

Reply:
Changed name to this